cypress/tsconfig.json

```
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
+    "lib": ["es2019", "dom"],
+    "types": ["cypress", "node"]
```

Please include`@testing-library/cypress` as well:
```suggestion
    "types": ["cypress", "node", "@testing-library/cypress"]
```